protocol/v2/qbft/spectest/msg_processing_type.go

```diff
@@ -67,7 +67,11 @@ func RunMsgProcessing(t *testing.T, test *spectests.MsgProcessingSpecTest) {
 	require.NoError(t, err)
 
 	// broadcasting is asynchronic, so need to wait a bit before checking
-	time.Sleep(time.Millisecond * 5)
+	select {
+	case <-t.Context().Done():
+		return
+	case <-time.After(5 * time.Millisecond):
```

Comment:
magic number, i would move that number to the top of the package + add some docs

Reply:
ah this one is from test, nvm